### ISSUE TITLE
For a formatter like `fmt_percent()` and `fmt_currency()` have option to put symbol just in first/top row

### ISSUE THREAD

**Issue by dalerobson427** [rstudio/gt#414]
In many tables, repeating units is redundant. Have the option at least to only put a unit in the first row. This might also be the first row of each row group (if there are row groups).

**Comment by rich-iannone** [comment]
Yes I see this quite commonly. This is a good feature request. Thanks!
